examples/CMakeLists.txt

```diff
@@ -253,11 +274,26 @@ install(FILES
         ${EXAMPLES_DIR}/bend/bend.yml
         ${EXAMPLES_DIR}/bend/bend.out.json
         ${EXAMPLES_DIR}/bend/mc_vs_md.agr
+        ${EXAMPLES_DIR}/gromosBend/gromosBend.yml
+        ${EXAMPLES_DIR}/gromosBend/gromosBend.out.json
+        ${EXAMPLES_DIR}/gromosBend/gromosBend-plot.py
+        ${EXAMPLES_DIR}/gromosBend/gromosBend-mc-rdf.dat
+        ${EXAMPLES_DIR}/gromosBend/dihedral-md-rdf.dat
         ${EXAMPLES_DIR}/dihedral/dihedral.yml
         ${EXAMPLES_DIR}/dihedral/dihedral.out.json
         ${EXAMPLES_DIR}/dihedral/dihedral-plot.py
         ${EXAMPLES_DIR}/dihedral/dihedral-mc-rdf.dat
         ${EXAMPLES_DIR}/dihedral/dihedral-md-rdf.dat
+        ${EXAMPLES_DIR}/improper-dihedral/improper-dihedral.yml
+        ${EXAMPLES_DIR}/improper-dihedral/improper-dihedral.out.json
+        ${EXAMPLES_DIR}/improper-dihedral/improper-dihedral-plot.py
+        ${EXAMPLES_DIR}/improper-dihedral/improper-dihedral-mc-rdf.dat
+        ${EXAMPLES_DIR}/improper-dihedral/improper-dihedral-md-rdf.dat
+        ${EXAMPLES_DIR}/fene_stretch/feneStretch.yml
```

Comment:
This is a minor comment, but I think it's better to use snake_case also for the files (as the other example). When installing all example files are placed in one directory.

Reply:
Fixed with ff815cc.